deployment/config/testnet/contracts.json

```diff
@@ -117,7 +117,7 @@
       "chainId": 6,
       "address": "0x2BCC362643B0aa3b2608de68b2C2ef2e6eFd2bb6",
       "constructorArgs": [
-        "0xC99E3A7BAaEb371b5eBEf714bA0b8b480359D168",
+        "0x1740211EbC1B7a25Ea97de6D12eEfCcd2A7CD7A7",
         "0xef14dd4a5b485ff12661450b1f0d1c1fd37db84eef14dd4a5b485ff12661450b1f0d1c1fd37db84e01ef14dd4a5b485ff12661450b1f0d1c1fd37db84e"
       ]
```

Comment:
Hm, I'm not sure we want to do this.

I think there are two separate use cases here:
- Preserving the original constructor arguments for easy contract verification
- Making available the constructor arguments that you would use right now to redeploy.

Since this is called `constructorArgs`, I think we should use this field for the first use case. Do we need something to address the second use case? :thinking: 
Is this used in the upgrade scripts for example?